prow/cmd/splice/main.go

```diff
@@ -62,6 +68,7 @@ func getQueuedPRs(url string) ([]int, error) {
 	queue := struct {
 		E2EQueue []struct {
 			Number int
+			Login  string
```

Comment:
Done. At one point I was going to include author somewhere, but decided against it.